tests/build/test_build_request.py

```diff
@@ -1348,3 +1348,85 @@ def test_prod_non_custom_base_image(self, tmpdir):
         pull_base_image_plugin = get_plugin(
             plugins, 'prebuild_plugins', 'pull_base_image')
         assert pull_base_image_plugin is not None
+
+    def test_render_prod_custom_site_plugin_enable(self):
+        """
+        Test to make sure that when we attempt to enable a plugin, it is
+        actually enabled in the JSON for the build_request post-rendering
```

Comment:
I meant that as literally "after running the render() method on build_request, it should be there" I can change the verbage if that's desired.